pyvista/plotting/camera.py

```diff
@@ -851,7 +868,10 @@ def tight(
 
         """
         # Inspired by vedo resetCamera. Thanks @marcomusy.
-        x0, x1, y0, y1, z0, z1 = self._renderer.bounds
+        if bounds is None:
+            x0, x1, y0, y1, z0, z1 = self._renderer.bounds
+        else:
+            x0, y0, x1, y1, z0, z1 = bounds
```

Comment:
To address the missing test coverage, we can probably cheat a bit and condense this to avoid the if-else branching.
```suggestion
        bounds = bounds if bounds is not None else self._renderer.bounds
        x0, y0, x1, y1, z0, z1 = bounds
```
If we add an example to the docstring, then we'll get the necessary test coverage from there.